elements/noflo-account.html

```diff
@@ -229,6 +272,31 @@
         logout: function () {
           this.fire('logout', true);
         },
+        relogin: function (event, details, sender) {
+          event.preventDefault();
+          var scope = sender.getAttribute('data-scope');
+          this.fire('relogin', scope);
+        },
+        userChanged: function () {
+          if (!this.user) {
+            return;
+          }
+          if (!this.user.plan || this.user.plan.type === 'free') {
```

Comment:
Storing `askForScope` in the widget introduces (more) statefulness. Is there at least a less stateful way to represent this transformation of user info to these scopes?
And is this the scopes we have asked for, or will ask for? The naming is not clear.